packages/cdktf-cli/src/bin/cmds/ui/get.tsx

```diff
@@ -55,7 +55,7 @@ export const Get = ({
         });
       } catch (e) {
         console.error(e);
-        exit(e);
```

Comment:
The ink exit callback accepts undefined and Error, a string being thrown will be ignored and treated as exit code 0. Wrapping the incoming Error in an Error makes it work